appdetect/git.go

```diff
@@ -51,42 +62,37 @@ func ScalingoRepoComplete(dir string) []string {
 	}
 
 	for _, remote := range remotes {
-		if strings.HasPrefix(remote.Name, "scalingo-") {
-			repos = append(repos, remote.Name[9:])
+		if strings.HasPrefix(remote.Config().Name, "scalingo-") {
+			repos = append(repos, remote.Config().Name[9:])
 		} else {
-			repos = append(repos, remote.Name)
+			repos = append(repos, remote.Config().Name)
 		}
 	}
 
 	return repos
 }
 
-func scalingoRemotes(directory string) (gitremote.Remotes, error) {
-	matchedRemotes := make(gitremote.Remotes, 0)
```

Comment:
Are you sure there is no regression here as the 'matchedRemotes' filter has been deleted?